4_advanced_usage/75_separate_database.txt

```diff
@@ -0,0 +1,106 @@
+=== Use a database on a separate server
+
+This section allows installing a separate databse only without splitting the rest of the server components
+like when using the rudder-multiserver-setup script. 
+The setup is done in two places: on the database server and on the Rudder root server.
+
+It also allows moving an existing database to another server.
+
+==== On the database server
+
+* Install and configure the agent on the node, and install the *rudder-reports* package.
+
+* Change the `postgresql.conf` file (usually in `/var/lib/pgsql` or `/etc/postgresql`), to listen on the right interface to communicate with the server:
+
+----
+
+listen_addresses = 'IP_TO_USE' 
+----
+
+* Add an authorization line for the server (in `pg_hba.conf`, in the same directory):
+
+----
+
+host    all         all         IP_OR_NETWORK_INCLUDING_THE_SERVER  trust
```

Comment:
Presumably we only want to allow the main server here, really. Can you clarify that, and make it we just put the ROOT_SERVER_IP?